packages/react-components/review/react-components.api.md

```diff
@@ -500,6 +527,9 @@ export interface TypingIndicatorStylesProps extends BaseCustomStylesProps {
     typingUserDisplayName?: IStyle;
 }
 
+// @public (undocumented)
+export const useIdentifiers: () => Identifiers;
```

Comment:
See comment about `IdentifierProvider`. This would also go away if the provider is removed from the API.

Reply:
The reason behind this approach is mentioned above.